code/src/NCModel/Managers.ts

```diff
@@ -24,6 +24,289 @@ export abstract class NcManager extends NcObject
     }
 }
 
+export class NcManufacturer extends BaseType
```

Comment:
Only suggestions are overall suggestion to start using TSDoc in the code as it is created since all the "documentation" is easier to add in when in context and will pay off in the future for understanding of the code (either for reader of the code direct or simple doc generation from the code using automatic tools